apps/sms/views/conversation/style/message.css

```diff
@@ -35,11 +40,15 @@
 }
 
 .message.incoming .bubble {
+  background: #39c3c2;
+}
+html[dir="ltr"] .message.incoming .bubble {
   float: left;
-
   border-radius: 0 1.5rem 1.5rem 1.5rem;
-
-  background: #39c3c2;
+}
+html[dir="rtl"] .message.incoming .bubble {
```

Comment:
Here in particular, or in general?

In general, that's a convention we took with the rest of the team. Basically, that's the best css equivalent of "document.dir", aka the direction of the language. The semantic of :-moz-dir is a bit different, and I'm pretty sure it is also useful is some case (but here, I don't see the difference).

I do like it because it protects your layout to be messed up if a `dir` is added in a parent element, including dir="auto". As I've seen some `dir = ltr` used in place of `text-align` or `float` or... quite a few times even in current master, I feel it is more robust this way. 

And it might be better performance-wise, because `:-moz-dir(rtl)` might need to walk the parents of current element until it found one with a 'dir' (I'm really not sure about that though. Maybe this info is available on the current node anyway).

Did I understand your question well?